front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -390,10 +393,24 @@ class AdminProjectEventEdit extends PureComponent<Props, State> {
   }
 }
 
-export default (props: InputProps) => (
-  <GetRemoteFiles resourceId={props.params.id} resourceType="event">
-    {(remoteEventFiles) => (
-      <AdminProjectEventEdit remoteEventFiles={remoteEventFiles} {...props} />
+const Data = adopt<Props>({
+  project: ({ params, render }) => (
+    <GetProject projectId={params.projectId}>{render}</GetProject>
```

Comment:
Bit confused by what's going on here, since the target branch has also been changed already (the folder structure is completely different from e.g. master). But why do we suddenly need the project data here? Or why didn't we need it before?

Reply:
The folder structure might be Brent's changes in his branch, changing the `projects/edit` to `projects/project`. I'm going to merge his branch into the feature branch and then point this one to there. That happened here: https://github.com/CitizenLabDotCo/citizenlab/pull/2026

For this line of code with the Data and `adopt`, I honestly don't know how it worked before. There was a `project` as a prop but I could not figure out where it was coming from. What I've changed it to works in manual testing. This is the only instance in the codebase I could find that had this issue.

Reply:
I think this is not needed. We use project only to get the projectId, but we can get that from the params.

Reply:
Yup, that's right. Fixed